src/components/RegistrationForm.js

```diff
@@ -0,0 +1,162 @@
+import React, { Component } from 'react'
+import { Redirect, Link } from 'react-router-dom';
+import { connect } from 'react-redux';
+import { userActions } from '../actions/userActions'
+
+class RegisterForm extends Component {
+    state = {
+        user: {
+            firstName: "",
+            secondName: "",
+            email: "",
+            password: "",
+            confirmedPassword: "",
+            birthday: "",
+            isEmployee: false
+        },
+        submitted: false,
+        isSamePasswords: false,
+
+    };
+    handleInputChange = (event) => {
+        const target = event.target;
+        const value = target.type === "checkbox" ? (target.checked ? target.checked : false) : target.value;
+        const name = target.name;
+        const { user } = this.state;
+
+        this.setState({
+            user: {
+                ...user,
+                [name]: value
+            }
+        });
+    }
+    handlesubmit = (e) => {
+        e.preventDefault();
+
+
+        this.setState({ submitted: true });
+        this.setState({ submitted: true });
+        const { user } = this.state;
+        const { dispatch } = this.props;
+
+        if (user.confirmedPassword !== user.password) {
+            this.setState({ isSamePasswords: false });
+            return;
+        }
+        this.setState({ 
+            isSamePasswords: true
+         });
+        if (user.firstName && user.secondName && user.email && user.password && user.confirmedPassword) {
+            const { confirmedPassword, ...userWithoutConfirm } = user;
+            dispatch(userActions.register(userWithoutConfirm));
+        }
+    }
+
+    render() {
+        const { registering, alert, registered, dispatch } = this.props;
+        const { user, submitted, isSamePasswords} = this.state;
+        if (registered) {
```

Comment:
"hasBeenRegistered" name is better, in my opinion.

Reply:
Done